js/packages/botSocket/botSocket.js

```diff
@@ -6,17 +6,16 @@
  * 2017
  */
 
-// native depensancies
+// depensancies
```

Comment:
Typo here

Reply:
nice

Reply:
I deleted the branch after I merged it (heh) I'm committing this to master